src/clj/puppetlabs/services/jruby/jruby_core.clj

```diff
@@ -204,11 +204,10 @@
   timeout. If the timeout runs out then nil will be returned, indicating that
   there were no instances available."
   [pool-context :- jruby-schemas/PoolContext
-   timeout :- schema/Int
```

Comment:
I'd say let's keep the change, update the docstring, and maybe add a TODO or comment saying that we should consider adding another arity in the future?